### PR TITLE
Update repo to explain maven artifact creation

### PR DESCRIPTION
**Pull request by erik-mythical**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+This project has only the protocol definitions for the SDK for inclusion as a submodule regardless of the language. The saga-sdk-proto maven
+artifact in the mytical central maven repository is built as part of the saga-sdk-java Java SDK client library. The maven artifact is used in
+conjuction with the Java SDK client library, but can also be used independently to provide the generated protocol buffer code for both Java
+and Kotlin. 
+
+### Updating the saga-sdk-proto maven artifact
+1. Clone or pull the saga-sdk-java repository.
+2. Do git submodule update if not already done with the --recursive flag when cloning
+3. Create feature branch
+4. Update the saga-proto submodule to the desired commit (most likely HEAD of develop)
```
Comment (axcrest) [inline README.md]: saga-sdk-proto

File: README.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+5. Update the Java SDK client so that the project can build successfully (proto changes will almost always require updates to the Java client)
+6. Bump the version for project_version variable in the root build.gradle
+7. When the changes are merged, the CI build will update the mythical central repo with the updated artifact
+
+### Releasing the saga-sdk-proto maven artifact
```
Comment (axcrest) [inline README.md:23]: Is the difference here just releasing to prod/master?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+
+## Building the saga-sdk-proto maven artifact
+
+This project has only the protocol definitions for the SDK for inclusion as a submodule regardless of the language. The saga-sdk-proto maven
+artifact in the mytical central maven repository is built as part of the saga-sdk-java Java SDK client library. The maven artifact is used in
+conjuction with the Java SDK client library, but can also be used independently to provide the generated protocol buffer code for both Java
+and Kotlin. 
+
+### Updating the saga-sdk-proto maven artifact
+1. Clone or pull the saga-sdk-java repository.
```
Comment (axcrest) [inline README.md]: Step 1-3 would be:
Clone this repo, saga-sdk-proto
Make your proto changes
After code reviews pass, commit those changes to develop 
 
Then Clone saga-sdk-java...right?

File: README.md (hunk 2 above)
Comment (erik-mythical) [reply README.md:23]: Mostly, this is a non-snapshot build so you can't overwrite it in the maven repo. Though once we are making the SDK public, the process will be different.

File: README.md (hunk 3 above)
Comment (erik-mythical) [reply README.md]: Yes. I'll clarify.

**Review by axcrest — APPROVED** [review]

**Pull request merged**